src/compiler/transformers/ts.ts

```diff
@@ -1815,14 +1826,15 @@ namespace ts {
             }
         }
 
+        type SerializedEntityNameAsExpression = Identifier | BinaryExpression | PropertyAccessExpression;

```

Comment:
nit, can you move this next to the other type alias. possibly move both to the top of the file.